be/src/formats/parquet/column_reader.cpp

```diff
@@ -156,7 +156,7 @@ void ColumnReader::get_subfield_pos_with_pruned_type(const ParquetField& field,
 
         auto it = field_name_2_pos.find(formatted_subfield_name);
         if (it == field_name_2_pos.end()) {
-            LOG(WARNING) << "Struct subfield name: " + formatted_subfield_name + " not found.";
+            DLOG(WARNING) << "Struct subfield name: " + formatted_subfield_name + " not found.";
```

Comment:
It is better to remove this debug log instead.

Reply:
Sure even better